src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderDamageObligationModel.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Builder;
+import lombok.Getter;
+
+import javax.validation.constraints.NotBlank;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+@Builder
+@Getter
+@ApiModel(description = "Damage obligation for an offender")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OffenderDamageObligationModel {
+
+    @ApiModelProperty(value = "Identifier of damage obligation", example = "1", position = 1)
+    private Long id;
+
+    @ApiModelProperty(value = "Offender number", example = "A12345", position = 2)
+    private String offenderNo;
+
+    @ApiModelProperty(value = "Reference number", example = "012388", position = 3)
+    private String referenceNumber;
```

Comment:
Is this a `reference` or a reference number?  The example is all digits but leading zero - good to know the correct format?

Reply:
This is indeed a reference number. I'll do some digging to figure out if there is a special format.